internal/proxy/providers/sso_test.go

```diff
@@ -241,8 +269,10 @@ func TestSSOProviderGetEmailAddress(t *testing.T) {
 				body, err := json.Marshal(tc.RedeemResponse)
 				testutil.Equal(t, nil, err)
 				p.RedeemURL, redeemServer = newTestServer(http.StatusOK, body)
+				p.ProxyRedeemURL, redeemServer = newTestServer(http.StatusOK, body)
 			} else {
 				p.RedeemURL, redeemServer = newCodeTestServer(400)
+				p.ProxyRedeemURL, redeemServer = newCodeTestServer(400)
```

Comment:
similar to @shrayolacrayon 's comment above, why do we need to create the test server twice here?